libs/application/templates/healthcare-license-certificate/src/fields/CustomFormConclusionSectionField/index.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+} from '@island.is/application/types'
+import { Box, Button, Text, PdfViewer } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC, useState } from 'react'
+import {
+  AlertMessageFormField,
+  MessageWithLinkButtonFormField,
+  PdfLinkButtonFormField,
+} from '@island.is/application/ui-fields'
+import { coreMessages, formatText } from '@island.is/application/core'
+import * as styles from './styles.css'
+import { confirmation } from '../../lib/messages'
+import { getProfessionName } from '../../utils/getProfessionName'
+import { conclusionMessages } from '@island.is/application/ui-forms'
+
+export const CustomFormConclusionSectionField: FC<
```

Comment:
Not sure id have to fiddle around with it myself before giving an answer but since there are so many reviews waiting I'll need to revisit this once the release has been completed. :D